src/pages/documentation/migrate-to-android.md

```diff
@@ -0,0 +1,89 @@
+| CampaignClassic | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/campaignclassic.svg?logo=android&logoColor=white&label=campaignclassic)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/campaignclassic)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+
+### Manual library import
+
+If you are manually importing the Mobile SDK libraries, update your app dependencies to the latest versions shown in the table above.
+
+### Gradle
+
+If you are using Gradle to manage your app dependencies, the following example shows how to upgrade to latest version in the `build.gradle` file.
+
+```java
+dependencies {
+    //replace sdk-core with Core/Lifecycle/Identity/Signal 2.0
+    //implementation 'com.adobe.marketing.mobile:sdk-core:1.+'
+    implementation 'com.adobe.marketing.mobile:core:2.+'
+    implementation 'com.adobe.marketing.mobile:identity:2.+'
+    implementation 'com.adobe.marketing.mobile:lifecycle:2.+'
+    implementation 'com.adobe.marketing.mobile:signal:2.+'
+
+    // replace UserProfile 1.+ with UserProfile 2.0
+    //implementation 'com.adobe.marketing.mobile:userprofile:1.+'
+    implementation 'com.adobe.marketing.mobile:userprofile:2.+'
+
+}
+```
+
+<InlineAlert variant="warning" slots="text"/>
+
+Using dynamic dependency versions is not recommended for production apps. Refer to this [page](https://github.com/adobe/aepsdk-core-android/blob/main/Documentation/MobileCore/gradle-dependencies.md) for managing Gradle dependencies.
+
+Save the `build.gradle` file and select `Sync Project with Gradle Files` in Android Studio to download the latet Android SDK.
+
+## Update SDK initialization
+
+After you have imported the new Android libraries, you'll need to update SDK initialization code as described below. With Mobile Core version 2.0.0 and above, the `MobileCore.start()` API is no longer required. The SDK has simplified initialization and registration of extensions by providing the `MobileCore.registerExtensions()` API. After the given extensions have been registered, the SDK will be initialized and the completion block will be executed. Code which used to reside in the start() block will now reside in the `MobileCore.registerExtensions()`'s completion block.
```

Comment:
... you'll need to update **the** SDK initialization code ...
**The code** which used to reside ...